src/test/resources/custom_fields.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
or maybe create a separate directory in test/resources? not sure yet... but the number of test resources keeps growing. it is hard to keep track of them all.